<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerTool</class>
 <widget class="QWidget" name="PlayerTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(67, 67, 67, 1);</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>14</y>
     <width>980</width>
     <height>6</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(201, 201, 201, 1);</string>
   </property>
   <property name="value">
    <number>7</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="PictureBox" name="icon_play" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="PictureBox" name="icon_backward" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="PictureBox" name="icon_forward" native="true">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="PictureBox" name="icon_sound" native="true">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>45</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>00:07/</string>
   </property>
  </widget>
  <widget class="QLabel" name="speed">
   <property name="geometry">
    <rect>
     <x>793</x>
     <y>40</y>
     <width>39</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>倍速</string>
   </property>
  </widget>
  <widget class="PictureBox" name="icon_setting" native="true">
   <property name="geometry">
    <rect>
     <x>914</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="PictureBox" name="icon_fullscreen" native="true">
   <property name="geometry">
    <rect>
     <x>968</x>
     <y>40</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="subtitle">
   <property name="geometry">
    <rect>
     <x>862</x>
     <y>40</y>
     <width>22</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>cc</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>45</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(153, 153, 153, 1)</string>
   </property>
   <property name="text">
    <string>02:16</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header location="global">picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
